apps/client/src/app/(home)/_components/ConvertingStoreButton/ConvertingStoreButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Flex, Text } from "@chakra-ui/react";
+import axios from "axios";
+import * as S from "./ConvertingStoreButton.style";
+
+export const ConvertingStoreButton = ({ setData }: { setData: (value: []) => void }) => {
+  const onStoreType = (type: "nearby" | "recommend") => {
+    // 퍼블리싱용
+    const getData = async () => {
+      const res = await axios.get("/api/products", { params: { storeType: type } });
+
+      const { result } = res.data;
+      setData(result);
+    };
+    getData();
+  };
+
+  return (
+    <Flex w="100%" position="absolute" left="0" bottom="0">
+      <S.StyledButton
+        w="100%"
+        columnGap="8px"
+        h="48px"
+        bgColor="black"
+        color="white"
+        onClick={() => onStoreType("nearby")}
+      >
+        <Box className="material-icons-outlined">local_mall</Box>
```

Comment:
```suggestion
    <Flex w="100%" position="absolute" left="0" bottom="0">
      <Button
        w="100%"
        columnGap="8px"
        h="48px"
        bgColor="black"
        color="white"
        borderRadius="0"
        onClick={() => onStoreType("nearby")}
      >
        <span className="material-icons-outlined" style={{ font-size: '16px' }}>local_mall</span>
```
따로 스타일 컴포넌트를 선언하신 이유가 있으실까요 ? borderRadius랑 아이콘 크기 조정은 위와 같이도 따로 스타일 컴포넌트 선언 없이도 가능할 거 같은데 혹시 어떻게 생각하시는 지도 궁금합니다~~

Reply:
material-icons-outlined를 사용할 때, 아이콘 크기를 조정하기 위해서 인라인 요소로 fontSize를 적용하기 적용이 되지 않아서 별도로 스타일드 컴포넌트를 두었습니다. className에 접근하면 적용이 되더라구요 !